vault/dendron.ref.config.md

```diff
@@ -242,3 +242,15 @@ It will work by adding it under the `dev` section (which is not created in defau
 dev:
     enableWebUI: true
 ```
+
+### enableExportPodV2
```

Comment:
would you mind updating the destination of the pull request to `dev` instead of `master`?  Other artifacts are being included like the pod stuff.